ansible/roles/oso_host_monitoring/templates/monitoring-config.yml.j2

```diff
@@ -82,15 +82,15 @@ zagg_client_config:
 # Generic Linux Checks
 host_monitoring_cron:
 - name: send pcp ping every 5 mintues
-  minute: "*/5"
+  minute: "*/15"
   job: ops-runner -f -s 60 -n cspp.pcp.ping /usr/bin/cron-send-pcp-ping
 
-- name: run pcp checks every minute
-  minute: "*"
+- name: run pcp checks every 5 minutes
+  minute: "*/5"
```

Comment:
Add comment:

# This is needed every minute for the performance team

Decision: Leave at 1 minute